### ISSUE TITLE
start with selector rather than a fragment?

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1386]
In #1338 we came to consensus that selectors are to be recommended and fragments are allowed ... but the implication on #1320 is that `start` SHOULD thus refer to a `SpecificResource` rather than a `Canvas`... which is not the current pattern.

The value is already an object with id and type rather than a URI, so we've already bitten off that level of verbosity.  Using a selector would avoid the semantic issue of whether uri#fragment is still a Canvas, rather than needing a new type... the type is SpecificResource.

This wouldn't be backwards incompatible, as startCanvas could previously only refer to the entire Canvas. 

#1338 is also inconsistent with #953.

**Comment by azaroth42** [comment]
```
{
  "start": {
    "type": "SpecificResource",
    "source": {
      "id": "https://example.org/iiif/id1/canvas1",
      "type": "Canvas"
    },
    "selector": {
      "type": "PointSelector",
      "conformsTo": "http://iiif.io/api/annex/annotation/pointSelector/",
      "value" : "tp=10"
  }
}
```

**Comment by azaroth42** [comment]
Decision from #1338 (etc) makes this uncontroversial.

**Comment by azaroth42** [comment]
Closed by #1500
